code/game/objects/items/toys.dm

```diff
@@ -211,6 +211,22 @@
 	attack_verb = list("attacked", "struck", "hit")
 	brightness_on = 0
 
+/obj/item/weapon/twohanded/dualsaber/toy/attack(target as mob, mob/living/user as mob)
+	if(HULK in user.mutations)
+		to_chat(user, "<span class='warning'>You grip the blade too hard and accidentally close it!</span>")
+		unwield()

```

Comment:
I could just remove this whole section but first though of the blade just closing on the clown instead. Just want to learn about the parent call first and then this can be adressed